chronograf/canned/TODO.go

```diff
@@ -1,5 +1,3 @@
-// +build !assets
```

Comment:
These 3 build flags are removed so that the placeholders are always in use - otherwise the build would fail because the `Makefile` doesn't recurse into `chronograf` anymore.